include/zephyr/net/wifi_utils.h

```diff
@@ -57,12 +57,12 @@ int wifi_utils_parse_scan_bands(char *scan_bands_str, uint8_t *band_map);
 
 
 /**
- * @brief Convert a string containing a list of SSIDs to an array of SSID strings.
+ * @brief Include a string containing a SSID to an array of SSID strings.
  *
- * @details The function will parse a string which specifies Wi-Fi SSIDs
- * as a comma separated string and convert it to an array.
+ * @details The function will add a string which specifies Wi-Fi SSID
+ * to an array of SSIDs.
```

Comment:
This is not really adding anything the brief didn't already mention so maybe just drop